types/styled-system/styled-system-tests.tsx

```diff
@@ -285,6 +288,7 @@ const Grid: React.ComponentType<GridComponentProps> = styled`
     ${gridAutoColumns};
     ${gridTemplateRows};
     ${gridTemplateColumns};
+    ${justifyItems};
```

Comment:
This should probably be added to [`Flex`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/247a03b2198d698e7283095f951aebf549705668/types/styled-system/styled-system-tests.tsx#L259) not in `Grid`